bedrock/firefox/firefox_details.py

```diff
@@ -223,8 +223,10 @@ def get_download_url(self, channel, version, platform, locale,
         # see bug 1324001
         if channel in ['alpha', 'nightly']:
             prod_name = 'firefox-nightly' if channel == 'nightly' else 'firefox-aurora'
-            # Use the stub installer for 32-bit Windows
-            if _platform == 'win' and locale == 'en-US' and not force_full_installer:
+            # Use the stub installer for approved platforms
+            stub_langs = settings.STUB_INSTALLER_LOCALES[channel].get(_platform, [])
```

Comment:
Do we know for a fact that 'alpha' and 'nightly' will always be in `settings.STUB_INSTALLER_LOCALES`? If not we should change the dict access to a `.get(channel, {})` to avoid an error in that case.